assessment.md

```diff
@@ -13,81 +13,512 @@
+        to each project
+    * E = Going above and beyond the expectations and contributions
+        of others
+  * Project management with GitHub, specifically the Flow Model
+    * N = A complete disregard for the model, especially multiple
+        instances of overriding the master branch
+    * I = Occasional disregard for the model including overriding
+        the master branch
+    * A = Almost always adhering to model, but never overriding
+        the master branch
+    * G = Making only minor mistakes while trying to follow the model
+        and providing a positive influence on other aspects such as issues
+        and communication
+    * E = A lack of mistakes in addition to emerging as a leader who
+      provides support to others who may be struggling with project
+      management
   * Continuous integration with Travis
-  * Foundations of software engineering
+    * N = Failing to enable Travis at all or not passing any requirements
+      on any assignment
+    * I = Enabling Travis, but failing the majority of checks
+    * A = Enabling Travis, and passing a majority of Travis checks
+    * G = Enabling Travis, and passing all of the Travis checks
+    * E = Enabling Travis, passing all checks, and assisting others who are
+      struggling or improving issues with Travis if found
+  * Foundations of software engineering practices
+    * N = Working separate from a team with no communication
+    * I = Working with a team a small amount of the time with no communication
+    * A = Working well with a team half of the time with little communication
+    * G = Working well with a team most of the time with clear communication
+    * E = Working well with a team all time with clear and constructive
+      communication
 * Understanding and avoiding red flags
+  * N = Ignoring all red flags
+  * I = Would be identifying red flags, but not fixing them
+  * A = Identifying and fixing a red flag
+  * G = Identifying, understanding and fixing a red flag to prevent future bugs
+  * E = Identifying, understanding, and fixing all red flags
 * Understanding and adopting best practices
+  * N = A severe lack of understanding and refusal to adopt best practices
+  * I = Slight understanding and occasional use and identification of the
+    best practices
+  * A = An average amount of understanding and general identification and use of
+      best practices
+  * G = Above average demonstrations of understanding and the ability to find and
+    utilize best practices
+  * E = Full understanding of concepts, would also include the ability to determine
+    which practices are better than others and then applying those practices
 
 ### Group
 
 * Attendance
+  * N = Missing five or more lab sessions
+  * I = Missing four lab sessions
+  * A = Missing three lab sessions
+  * G = Missing two lab sessions
+  * E = Missing one or less lab sessions
 * Deadlines
+  * N = If an assignment is turned in 5 or more days past the deadline
```

Comment:
Levels that are the same should not be repeated. Use some means of reducing this redundancy.

Reply:
I totally agree with that and I will take a look to see what can I revise